### PR TITLE
To add state dict and load_dict for Chained Scheduler

### PR DESCRIPTION
**Pull request by iramazanli**
Unblocks https://github.com/pytorch/vision/issues/4281

Adding state_dict() and load_state_dict() methods for Chained Scheduler

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, thanks!
